Justfile

```diff
@@ -17,5 +17,11 @@
 # limitations under the Licenses.
 
 keysyms:
-    cargo run --manifest-path keysym-generator/Cargo.toml \
-        src/automatically_generated.rs
\ No newline at end of file
+    docker container run --rm \
+        --name keysym_generator \
+        --mount type=bind,source="$(pwd)",target=/xkeysym \
+        -it rust:slim-bookworm \
+        sh -c "apt-get update && apt-get install x11proto-core-dev && \
+        cargo run --manifest-path /xkeysym/keysym-generator/Cargo.toml \
+           /xkeysym/src/automatically_generated.rs"
```

Comment:
We should document this somewhere in a section possibly named `Updating headers`